canonical-web-and-design/ubuntu.com
# Integrate cypress

## Done

- Integrate cypress in ubuntu.com test suite
- Add first simple test
- Add configuration to avoid cypress to generate default files

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- In another terminal run `PORT=8001 yarn run cypress:run`
- Make sure tests pass.
- In another terminal run `PORT=8001 yarn run cypress:open`
- Have fun with the interface that can run the tests that cypress provides

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/2361
